### PR TITLE
Hints for Sleepy Cow Sorting

### PR DESCRIPTION
**Pull request by brad182**
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.

### CODE REVIEW

File: solutions/bronze/usaco-892.mdx [hunk 1]
```diff
@@ -14,6 +14,22 @@ By Vikas Thoutam
 
 <Spoiler title="Video Solution Code">
 
+<Spoiler title="Hint 1">
+
+Farmer John can only give orders to a cow at the front of the line,
+so would you need to go through all of the cows before a later cow
+to get to it?
```
Comment (SansPapyrus683) [inline solutions/bronze/usaco-892.mdx:21]: what's "it"

**Comment by SansPapyrus683** [comment]
brad can you please address

File: solutions/bronze/usaco-892.mdx [hunk 2]
```diff
@@ -14,6 +14,22 @@ By Vikas Thoutam
...
+so would you need to go through all of the cows before a later cow
+to get to that later cow?
+
+</Spoiler>
+
+<Spoiler title="Hint 2">
+
+If a group of cows reaching the end of the list of cows, is already sorted,
+is it necessary to keep giving orders when you reach the first cow in this
+group?
```
[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply solutions/bronze/usaco-892.mdx:29]: bump

[the remark this answers is not in the document]
Comment (brad182) [reply solutions/bronze/usaco-892.mdx:29]: i mean it is a hint so do you need to give that much information?

[the remark this answers is not in the document]
Comment (SansPapyrus683) [reply solutions/bronze/usaco-892.mdx:29]: but when the user doesn't even know what the hint is talking about...

**Pull request closed without merging**